website/content/docs/api-clients/client-agent.mdx

```diff
@@ -652,6 +652,17 @@ Learn more: https://cfl.re/CF_DNS_PROXY_FAILURE
 You can still install both the Cloudflare WARP client and the Boundary Client Agent on the same machine.
 As long as you don't run both at the same time, they should work as expected.
 
+### Huntress EDR client
+
+Host isolation and release via Huntress does not work as expected in Windows when used alongside the Boundary Client Agent.
```

Comment:
```suggestion
Host isolation and release do not work as expected in Windows when you use the Huntress EDR client with the Boundary Client Agent.
```

Reply:
We try to avoid using Latin words like "via" since they may not be familiar to some readers. I also recommend changing  "alongside" to "with" since it's shorter and more common.